viam-cartographer.go

```diff
@@ -133,7 +128,8 @@ func New(
 		return nil, err
 	}
 
-	cancelCtx, cancelFunc := context.WithCancel(ctx)
+	// Need to pass in a long-lived context because ctx is short-lived
+	cancelCtx, cancelFunc := context.WithCancel(context.TODO())
```

Comment:
I verified that the orb module os process terminates (either immediately or after a few seconds) regardless of whether viam-server is killed via `SIGTERM` `SIGQUIT` or `SIGKILL`.

Reply:
sry, I posted on the wrong PR. Ignore this comment